draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -765,35 +765,64 @@ Otherwise, the processed SD-JWT payload can be passed to the application to be u
+For both the General and Flattened JSON Serialization, the SD-JWT is represented
+as a JSON object according to Section 7.2 of [@!RFC7515]. The following new
+unprotected header parameters are defined:
 
-Verification of the JWS JSON serialized SD-JWT follows the same rules defined in (#verification),
-except that the SD-JWT does not need to be split into component parts and the Disclosures
-can be found in the respective member of the JSON object (or elsewhere).
+ * `disclosures`: An array of strings where each element is an individual
+   Disclosure as described in (#creating_disclosures).
+ * `kb_jwt`: A Key Binding JWT as described in (#kb-jwt).
 
-Using a payload similar to that from [Example 1](#example-1), the following is a non-normative example of
-a JWS JSON serialized SD-JWT from an Issuer with all the respective Disclosures.
+If a Key Binding JWT is present, the digest in the `sd_hash` claim MUST be taken
+over a string built as described in (#integrity-protection-of-the-presentation).
+The "Issuer-signed JWT" part is built by concatenating the protected header, the
+payload, and the signature of the JWS JSON serialized SD-JWT using a `.`
+character as a separator, and using the Disclosures from the `disclosures`
+member of the unprotected header. In case of multiple signatures, only the first
+one is used for the Disclosures and Key Binding JWT.
 
-<{{examples/json_serialization/sd_jwt_issuance.json}}
+## Flattened JSON Serialization
 
-Below is a non-normative example of a presentation of the JWS JSON serialized SD-JWT, where the Holder
-has selected to disclose `given_name`, `family_name`, and `address`.
+In case of the Flattened JSON Serialization, there is only one unprotected
+header.
 
-<{{examples/json_serialization/sd_jwt_presentation.json}}
+The following is a non-normative example of a JWS JSON serialized SD-JWT as
+issued using the Flattened JSON Serialization:
 
+<{{examples/json_serialization_flattened/sd_jwt_issuance.json}}
+
+The following is a presentation including a Key Binding JWT and two Disclosures:
+
+<{{examples/json_serialization_flattened/sd_jwt_presentation.json}}
+
+## General JSON Serialization
+
+In case of the General JSON Serialization, there are multiple unprotected
+headers (one per signature). If present, `disclosures` and `kb_jwt`, MUST be
+included in the first unprotected header and MUST NOT be present in any
+following unprotected headers.
+
+The following is a non-normative example of a presentation of a JWS JSON
+serialized SD-JWT including a Key Binding JWT using the General JSON
+Serialization:
+
+<{{examples/json_serialization_general/sd_jwt_presentation.json}}
+
+## Verification of the JWS JSON Serialized SD-JWT
+
+Verification of the JWS JSON serialized SD-JWT follows the same rules defined in
+(#verification), except for the following aspects:
+
+ * The SD-JWT does not need to be split into component parts and the Disclosures
+   can be found in the respective member of the JSON object.
```

Comment:
Disclosures can be found in the unprotected header, no? I am probably just not reading this right..

Reply:
```suggestion
   can be found in the respective member of the JSON object in the unprotected header.
```